tests/it/fossa_cli.rs

```diff
@@ -96,6 +96,35 @@ async fn analyze_fails() {
     );
 }
 
+/// Analysis of a dynamic-only project should fail, since dynamic strategies are disabled.
+/// Rust is dynamic only, so analyze Broker itself.
+#[tokio::test]
+async fn analyze_fails_dynamic() {
+    let (_tmp, config, ctx) = temp_config!(load);
+    let scan_id = Uuid::new_v4().to_string();
+
+    // Provide a path that doesn't exist, so that analysis fails.
+    let project = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+
+    println!("Downloading CLI");
+    let location = fossa_cli::download(&ctx, config.debug().location(), DesiredVersion::Latest)
+        .await
+        .expect("must download CLI");
+
+    // Scan our project.
+    println!("Analyzing '{}' with scan id '{scan_id}'", project.display());
+    let err = location
+        .analyze(&scan_id, &project)
+        .await
+        .expect_err("must fail to analyze");
+
+    // Snapshot testing won't work here as the order of log lines output by FOSSA CLI are not determinstic.
+    // Just check for some specific substrings in the error output.
+    let rendered = format!("{err:#}");
+    assert!(rendered.contains("No analysis targets found in directory."));
+    assert!(rendered.contains("Could not find executable: `cargo`."));
```

Comment:
It looks like this assertion is not true on Windows (based on looking at the failing CI)

Reply:
Turns out FOSSA CLI just passes through the error and it's different on Windows 😭 just dropping this part of the check.